src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+
+Since you're experiencing a spike in errors in the `checkout-service` application and the `Sinatra/Rubytron/POST #purchaseCart` transaction, it's helpful to compare those errors with your throughput results. 
+
+If your throughput is very high during a time of service degradation it could indicate that your application is processing more work than it can handle. On the other hand, low throughput could indicate that your app isn't handling very many requests. This could simply mean that it's not used very much or it could mean that a service upstream that calls to your application is broken and requests aren't being processed. 
+
+<img
+    title="investigate throughput"
+    alt="A gif that shows you how to investigate throughput in the transaction details page"
+    src={apmInvestigateThroughputinAPM}
+/>
+
+</Step>
+
+<Step>
+
+### Investigate your transaction breakdown
+
+The transaction breakdown refers to the detailed breakdown of the time spent on each operation within a specific transaction. This breakdown helps developers understand the performance of the application and identify bottlenecks or areas for improvement.
+
+You can sort the transaction breakdown by **Category**, **% Time**, **Avg Calls/Txn**, and **Avg Time**. We recommend sorting by **% Time** to understand which piece of the transaction is taking up the most time within each transaction. 
+
+Let's examine the sections of the **Breakdown table** in the screenshot below:
+
+* **Category** refers to the type of call segment being monitored. 
+* **Controller** is a type of **category** that refers to model-view-controller architecture and is most likely a call made directly from your UI to your application. 
+* **External** is a type of **category** that represents a call being made that we began monitoring before we know where it's going.  
+*  **Middleware** is a type of **category** that most often refers to a dependency or library that is being used as part of your application's functionality. 
+* **Segment** consists of the name of the function and call segment being monitored.
+* **% Time** refers to the percentage of time each specific segment takes of the total transaction lifecycle. It's important to remember that some segments wait until all the other segments are completed before they're completed. In the screenshot below, the `Controller` segment is what is initiating the `POST` call, which means that all the other transactions are _nested_ within the `POST` call, which could be why it's taking longer than all the others. 
+* **Avg Calls/Txn** is the amount of times that the segment is called per transaction. 
+* **Avg Time** is the average time that a particular segment took to complete. It's important to keep in mind that there could be asynchronous work happening. One segment could be ongoing and then other, nested work, is being done at the same time. 
+
+Once you sort the **Breakdown table**, you can click on any problematic transactions. 
```

Comment:
think it's better to just word these references to UI informally, like "once you sort the breakdown table..."